contest_runner.py

```diff
@@ -319,64 +319,79 @@ def upload_file(file_full_path, remote_name=None, remove_local=False):
     def store_results(self):
         # Basic data stats
         data_stats = {
-            "games": self.games,
-            "team_stats": self.team_stats,
-            "random_layouts": [l for l in self.layouts if l.startswith("RANDOM")],
-            "fixed_layouts": [l for l in self.layouts if not l.startswith("RANDOM")],
-            "max_steps": self.max_steps,
-            "organizer": self.organizer,
-            "timestamp_id": self.contest_timestamp_id,
+            'games': self.games,
+            'team_stats': self.team_stats,
+            'random_layouts': [l for l in self.layouts if l.startswith('RANDOM')],
+            'fixed_layouts': [l for l in self.layouts if not l.startswith('RANDOM')],
+            'max_steps': self.max_steps,
+            'organizer' : self.organizer,
+            'timestamp_id' : self.contest_timestamp_id
         }
 
-        # Process replays: compress and upload
-        replays_archive_name = "replays_%s.tar" % self.contest_timestamp_id
-        replays_archive_name += ".gz" if self.compress_logs else ""
-        replays_archive_full_path = os.path.join(
-            self.replays_archive_dir, replays_archive_name
-        )
-        with tarfile.open(
-            replays_archive_full_path, "w:gz" if self.compress_logs else "w"
-        ) as tar:
-            tar.add(self.tmp_replays_dir, arcname="/")
+        # PROCESS REPLAYS: compress and upload
+        replays_archive_name = 'replays_%s.tar' % self.contest_timestamp_id
+        replays_archive_name += '.gz' if self.compress_logs else ''
+        replays_archive_full_path = os.path.join(self.replays_archive_dir, replays_archive_name)
+        with tarfile.open(replays_archive_full_path, 'w:gz' if self.compress_logs else 'w') as tar:
+            tar.add(self.TMP_REPLAYS_DIR, arcname='/')
```

Comment:
Here check because I don't think we use `self.TMP_REPLAYS_DIR` anymore. 

I think it is `self.tmp_replays_dir` as it is not a constant

Reply:
Yes this was true, fixed when merged from master!